perfmetrics/scripts/continuous_test/ml_tests/tf/continuous.cfg

```diff
@@ -0,0 +1,3 @@
+build_file: "gcsfuse/perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh"
+# Timeout so that it runs for 14 days
```

Comment:
Discussed with Prince, we should keep it 16 days and run the model for 14-15 days.